src/super_gradients/recipes/cityscapes_stdc_seg50.yaml

```diff
@@ -52,44 +52,42 @@ dataset_params:
   _convert_: all
   batch_size: 8
   val_batch_size: 8
-  color_jitter: 0.5
-  random_scales:
-    - 0.125
-    - 1.5
   crop_size:
     - 1024
     - 512
   eval_scale: 0.5
   image_mask_transforms_aug:
-    _target_: torchvision.transforms.Compose
-    transforms:
-      - _target_: super_gradients.training.utils.segmentation_utils.ColorJitterSeg
-        brightness: ${dataset_params.color_jitter}
-        contrast: ${dataset_params.color_jitter}
-        saturation: ${dataset_params.color_jitter}
+    Compose:
```

Comment:
Why do we need Compose -> transforms? Can it be handled in the factory class, and not in YAML?

Reply:
to keep the general logic of all factories:

CalssName:
    parameter_name: parameter_value